collect/cache/cache.go

```diff
@@ -67,127 +68,94 @@ func NewInMemCache(
 		met.Register(metadata)
 	}
 
-	if capacity == 0 {
-		capacity = DefaultInMemCacheCapacity
+	// if using the default capacity, allow the cache to grow really large by using math.MaxInt32 (2147483647)
+	if capacity == DefaultInMemCacheCapacity {
```

Comment:
I'm not sure if we still want to allow customers to configure CacheCapacity anymore. If we do, we need to also support the trace ejection case from the cache

Reply:
Yeah, true. I was trying to think what additional effects it would have as incoming and peer queues are based on this config value. I think you're right though, the cache doesn't be user configurable.

Reply:
Updated in https://github.com/honeycombio/refinery/pull/1359/commits/06023d02593395c1fa96f92e6865c0cb2a2802aa.